### PR TITLE
feat: allow scoping transfers by txis

### LINKED ISSUE #307: internal transfers should also be scope queried by txi

### PR DESCRIPTION
**Pull request by sborrazas**
closes #307

### CODE REVIEW

**Review by thepiwo — APPROVED** [review]

File: lib/ae_mdw/transfers.ex [hunk 1]
```diff
@@ -54,6 +56,18 @@ defmodule AeMdw.Transfers do
...
 
+  defp inside_txi_range?(
+         {:txi, %Range{first: first_txi, last: last_txi}},
+         {_gen_txi, _kind, _account_pk, ref_txi}
+       )
+       when first_txi < last_txi,
+       do: not is_nil(ref_txi) and first_txi <= ref_txi and ref_txi <= last_txi
+
+  defp inside_txi_range?({:txi, %Range{last: last_txi}}, {_gen_txi, _kind, _account_pk, ref_txi}),
+    do: not is_nil(ref_txi) and last_txi <= ref_txi and ref_txi >= last_txi
```
Comment (jyeshe) [inline lib/ae_mdw/transfers.ex:67]: `last_txi <= ref_txi and ref_txi >= last_txi -> last_txi == ref_txi`

could we have a test case for this situation too?

**Review by jyeshe — COMMENTED** [review]
Just test coverage.

File: lib/ae_mdw/transfers.ex (hunk 1 above)
Comment (sborrazas) [reply lib/ae_mdw/transfers.ex]: nice catch! I've fixed the code and added a test case for it

**Comment by thepiwo** [comment]
let get this reviewed and merged to be in our next release :)

**Review by jyeshe — APPROVED** [review]
:rocket:

**Pull request merged**